vllm/model_executor/layers/quantization/utils/marlin_utils.py

```diff
@@ -91,6 +91,18 @@ def marlin_make_workspace(output_size_per_partition: int,
                        requires_grad=False)
 
 
+def marlin_is_k_full(act_order: bool, is_row_parallel: bool) -> bool:
+    return (act_order and not is_row_parallel)
```

Comment:
The condition here is correct, but it would be better to pass is_k_full for the case where act_order == False.
Like this:
`return (not act_order) or (act_order and not is_row_parallel)`